ui/src/components/icon/QIcon.js

```diff
@@ -172,13 +175,22 @@ export default Vue.extend({
       data.attrs.focusable = 'false' /* needed for IE11 */
       data.attrs.viewBox = this.type.viewBox
 
+      // append attrs as style
+      if (this.type.attrs !== undefined) {
+        console.log(this.type.attrs)
+        let attrs = []
+        while ((attrs = REGEX_ATTRS.exec(this.type.attrs)) !== null) {
+          data.style[attrs[1]] = attrs[2]
```

Comment:
It would be better if we just read the style as a String and assign it as such (data.style = styleAsString), rather than splitting them individually.

Reply:
That was the initial attempt. However, when used as "styles=" the QIcon styling overwrote it. I have a video I can show you with the issue.